### PR TITLE
Move widgets to hmi_widgets

### PR DESCRIPTION
**Pull request by Minyewoo**

### CODE REVIEW

File: lib/src/charts/crane_load_chart/swl_data.dart [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+    required TextFile yCsvFile,
+    required List<TextFile> swlCsvFiles,
   }) :
-    _assetPath = assetPath,
-    _count = count;
+    _xCsvFile = xCsvFile,
+    _yCsvFile = yCsvFile,
+    _swlCsvFiles = swlCsvFiles;
   ///
   List<double> _parseStringList(List<String> strings) {
```
Comment (a-givertzman) [inline lib/src/charts/crane_load_chart/swl_data.dart:21]: I propose such an addition:
```
  List<double> _parseStringList(List<String> strings) {
    return strings
      .where((string) => string.trim().isNotEmpty)
```

File: test/unit/charts/crane_load_chart/swl_data/swl_data_test.dart [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hmi_core/hmi_core.dart';
+import 'package:hmi_widgets/src/charts/crane_load_chart/swl_data.dart';
+import 'fake_text_file.dart';
+void main() {
+  Log.initialize(level: LogLevel.off);
+  group('SwlData', () {
+    test('creates normally with valid files', () async {
+      const swlTestData = [
+        {
```
Comment (a-givertzman) [inline test/unit/charts/crane_load_chart/swl_data/swl_data_test.dart:10]: my suggestion:
```
        {
          'text_x': '1,1;2,7;3,2;4;5\n \n6,1;7,2;8,3;9,4;10,5',
          'text_y': '6,1;7,2;8,3;9,4;10,5\n\n\n1,1;2,7;3,2;4;5',
          'text_swl': [
            '1,1;2,7;3,2;4;5\n   \n6,1;7,2;8,3;9,4;10,5',
            '\n6,1;7,2;8,3;9,4;10,5\n1,1;2,7;3,2;4;5\n',
            '1,1;2,7;3,2;4;5\n\n \n6,1;7,2;8,3;9,4;10,5',
          ],
          'x': [1.1, 2.7, 3.2, 4.0, 5.0, 6.1, 7.2, 8.3, 9.4, 10.5],
          'y': [6.1, 7.2, 8.3, 9.4, 10.5, 1.1, 2.7, 3.2, 4.0, 5.0],
          'swl': [
            [1.1, 2.7, 3.2, 4.0, 5.0, 6.1, 7.2, 8.3, 9.4, 10.5],
            [6.1, 7.2, 8.3, 9.4, 10.5, 1.1, 2.7, 3.2, 4.0, 5.0],
            [1.1, 2.7, 3.2, 4.0, 5.0, 6.1, 7.2, 8.3, 9.4, 10.5],
          ],
        },
```

**Review by a-givertzman — COMMENTED** [review]
would you like to apply my suggestions?

File: lib/src/edit_field/network_dropdown_field/oil_data.dart [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+///
+/// Reads oil names and spec data for [SettingsPage] - HPU 
+/// from assets json file
+class OilData {
+  static const _debug = true;
+  final String _assetName;
+  final Map<String, Map<String, dynamic>> _data =  {};
+  ///
+  OilData({
+    required String assetName,
```
[the remark this answers is not in the document]
Comment (Minyewoo) [reply lib/src/edit_field/network_dropdown_field/oil_data.dart]: Already using. Probably you look at the old commit.

**Review by a-givertzman — APPROVED** [review]

**Pull request merged**